tests/test_indexes.py

```diff
@@ -25,27 +25,6 @@
 
 if TYPE_CHECKING:
     MYPY_CHECKING: bool = True
-    # See test_sorted_and_list() where mypy and pyright do different
-    # inference on sorted(pd.Index)
-    if MYPY_CHECKING:
-        from typing import Any
-
-        from typing_extensions import TypeAlias
-
-        SupportsRichComparison: TypeAlias = Any
```

Comment:
probably still need this in there??

Reply:
Not at the moment, that was only needed because the item type of list was Any/Unknown which created a discrepancy between mypy and pyright.